pttrack/models.py

```diff
@@ -71,8 +71,9 @@ def make_filepath(instance, filename):
     while carry_on:
         new_filename = "%s.%s" % (User.objects.make_random_password(48),
                                   filename.split('.')[-1])
-        path = '/'.join([instance.__class__.__name__.lower(),
-                         field_name, new_filename])
+        #path = '/'.join([instance.__class__.__name__.lower(),field_name, new_filename])
+
+        path = new_filename
```

Comment:
It feels a bit hacky to me to leave it that way (what if we want to serve other media type things in the future?), but at the moment this permissions thing is such a pain I would suggest we just call it for this one and don't worry about it.